$NetBSD: patch-Makefile,v 1.1.1.1 2011/03/05 17:54:35 brook Exp $

Use more generic names for the installed files and ensure that rpath
information is correctly included within the shared library.

--- Makefile.orig	2009-03-15 10:02:47.000000000 -0600
+++ Makefile	2011-02-12 10:59:40.000000000 -0700
@@ -1,9 +1,7 @@
 
-MODULE_big = uuid-freebsd
-OBJS = uuid-freebsd.o
-DATA_built = uuid-freebsd.sql
-DATA = uninstall_uuid-freebsd.sql
-SHLIB_LINK = -lmd
+MODULES = uuid
+DATA_built = uuid.sql
+DATA = uninstall_uuid.sql
 
 PG_CONFIG = pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
